src/Microsoft.Diagnostics.Monitoring.WebApi/Models/EventPipeProvider.cs

```diff
@@ -15,19 +18,19 @@ public class EventPipeProvider
     {
         [JsonPropertyName("name")]
         [Required]
-        public string Name { get; set; }
+        public string Name { get; set; } = string.Empty;
 
         [JsonPropertyName("keywords")]
 #if !SCHEMAGEN
         [IntegerOrHexString]
 #endif
-        public string Keywords { get; set; } = "0x" + EventKeywords.All.ToString("X");
+        public string? Keywords { get; set; } = "0x" + EventKeywords.All.ToString("X");
```

Comment:
Is this really nullable given that it has a default non-null value?

Reply:
From an `openapi.json` perspective it is since there's no `[Required]` attribute (and we have null handling for it).

Practically though, it should be marked as non-nullable and `[Required]` since we throw an exception if it is set to null.


With all of that said, I'm trying to avoid any changes to `openapi.json`, even if it doesn't seem right. This is something we should clean up seperately.

Reply:
I'll create an issue to track these sorts of problems